images/macos/toolsets/toolset-13.json

```diff
@@ -72,7 +72,6 @@
             "p7zip",
             "packer",
             "perl",
-            "pkg-config",
             "swiftformat",
             "tcl-tk",
```

Comment:
What will be changed if we just use a package with a version here instead of another condition? 🤔

Reply:
We could update as "tcl-tk@8" directly here as well..just thought this would be neat and similar format to earlier pinned versions.

Reply:
> earlier pinned versions

Ah, we simply didn't have the opportunity to pin it normally through the toolset - there are edge cases or methods that you can't call popular. It's better to control versioning through the toolset - it's the most visual file.